snorkel/labeling/model/label_model.py

```diff
@@ -280,6 +280,15 @@ def _init_params(self) -> None:
         # Handle single values
         if isinstance(self.train_config.prec_init, (int, float)):
             self._prec_init = self.train_config.prec_init * torch.ones(self.m)
+        elif isinstance(self.train_config.prec_init, np.ndarray):
+            self._prec_init = torch.from_numpy(self.train_config.prec_init)
+        elif isinstance(self.train_config.prec_init, list):
+            self._prec_init = torch.Tensor(self.train_config.prec_init)
+        elif not isinstance(self.train_config.prec_init, torch.Tensor):
+            print("TYPE ERROR")
```

Comment:
Looks like we forgot to remove this

Reply:
If you mean replacing `torch.from_numpy` with `torch.Tensor`, then this should be resolved now.

Reply:
I was referring to removing the `print` statement

Reply:
Yes, oops. Had forgotten to remove it.